pkg/eden/eden.go

```diff
@@ -339,6 +339,47 @@ func StatusEServer() (status string, err error) {
 	return state, nil
 }
 
+//StartEVEVBox function run EVE in VirtualBox
+func StartEVEVBox(vmName string, vBoxCpus int, vBoxMem int, hostFwd map[string]string) (err error) {
+
+	commandArgsString := fmt.Sprintf("startvm  %s", vmName)
+	if err = utils.RunCommandWithLogAndWait("VBoxManage", defaults.DefaultLogLevelToPrint, strings.Fields(commandArgsString)...); err != nil {
```

Comment:
Seems, that we need to fire `startvm` again in case of new VM after `createvm` and `modifyvm`.